### ISSUE TITLE
Release testdat 0.3.0

### ISSUE THREAD

**Issue by gorcha** [socialresearchcentre/testdat#47]
First release:

* [x] `usethis::use_cran_comments()`
* [x] Proof read `Title:` and `Description:`
* [x] Check that all exported functions have `@returns` and `@examples`
* [x] Check that `Authors@R:` includes a copyright holder (role 'cph')
* [x] Check [licensing of included files](https://r-pkgs.org/license.html#code-you-bundle)
* [x] Review <https://github.com/DavisVaughan/extrachecks>

Prepare for release:

* [x] `devtools::build_readme()`
* [x] [`urlchecker::url_check()`](https://github.com/r-lib/urlchecker)
* [x] `devtools::check(remote = TRUE, manual = TRUE)`
* [x] `devtools::check_win_devel()`
* [x] `rhub::check_for_cran()`
* [x] Review pkgdown reference index for, e.g., missing topics

Submit to CRAN:

* [x] `usethis::use_version('minor')`
* [x] `devtools::submit_cran()`
* [x] Approve email

Wait for CRAN...

* [x] Accepted :tada:
* [x] `usethis::use_github_release()`
* [x] `usethis::use_dev_version()`
* [x] Update install instructions in README

**Comment by kinto-b** [comment]
@gorcha Is there anything else you think we need to do before we can run the remaining checks and submit?

**Comment by gorcha** [comment]
I think just `chk_pattern()` - I've already done this, just need to push. Shall do a bit later tonight then I think we're good to go!

**Comment by kinto-b** [comment]
Woo 🎉

**Comment by gorcha** [comment]
Submitted! :champagne:

**Comment by gorcha** [comment]
Accepted :tada:

@kinto-b thanks for all your hard work!
Can you please do the notification thingos that you did for makepipe (r-weekly etc.)?

**Comment by kinto-b** [comment]
Stellar!!

Yup, I'll do a PR on R weekly now and I'll send an email around DPA on Monday.

**Comment by gorcha** [comment]
Not really sure where else. The [rOpenSci guide](https://devguide.ropensci.org/marketing.html) basically just says R weekly.

Possibly should put in a request for the [CRAN Reproducible Research task view](https://cran.r-project.org/web/views/ReproducibleResearch.html). You should definitely hit them up about makepipe.

If you could do the R Weekly bit I can check out other stuff. Thanks!

**Comment by kinto-b** [comment]
Oh good one. Yeah, I've added an issue to suggest inclusion of makepipe

Have made a PR for R Weekly here: https://github.com/rweekly/rweekly.org/pull/994

**Comment by gorcha** [comment]
Awesome, thanks!
